tutorials/epochs/baseline_regression.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-baseline_regression:
+
+====================================================================
+Baseline correction using generalized linear mixed models
+====================================================================
+
+By including the baseline interval as a regressor in a generalized
+linear model, we allow the data to determine how much baseline
+correction is necessary for the specific epoch. This approach can
+increase statistical power (Alday et al., 2019).
+Alday et al. provided a script at osf: https://osf.io/85bjq
```

Comment:
Let's try to make this introduction a bit more verbose. Something like this maybe?

```
Traditional baseline correction simply adds or subtracts a scalar amount from every
timepoint in an epoch, such that the mean value during the defined baseline period 
is zero.  This can be bad because [...] :footcite:`Alday2019`. Here we demonstrate 
the alternative regression-based method recommended in that publication, where the 
*strength of the effect* of the baseline period is allowed to vary across each 
timepoint of the epoch. 
```

(you can fill in the `[...]`)

note the `footcite` formatting I've suggested; you can `git grep footbibliography` (or just look at one of the other tutorial source files) to see how it's handled. You'll also need to add a BibTeX entry into `doc/references.bib` for the Alday paper.